cli_tools_e2e_test/common/logging/logging.go

```diff
@@ -0,0 +1,69 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package logging
+
+import (
+	"bufio"
+	"bytes"
+	"io"
+	"log"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
+)
+
+// AsDaisyLogger returns a daisy.Logger that writes to a log.Logger.
+func AsDaisyLogger(logger *log.Logger) daisy.Logger {
+	return goLogToDaisyLog{logger: logger}
+}
+
+// AsWriter returns an io.Writer that writes to a log.Logger.
+func AsWriter(logger *log.Logger) io.Writer {
+	return goLogToIoWriter{logger: logger}
+}
+
+// goLogToDaisyLog is an implementation of daisy.Logger that writes to a log.Logger backend.
+type goLogToDaisyLog struct {
```

Comment:
shouldn't the name be reversed, daisy logs (API) writing to Go log output?

Reply:
The idea is that it *converts* a Go log to a Daisy log -- given a concrete `log.Logger`, you get an object that you can use wherever a daisy logger is required.

Reply:
Still seems backwards to me, but the docs should clear any confusion